kedro/framework/cli/project.py

```diff
@@ -360,6 +369,13 @@ def activate_nbstripout(
     help=TAG_ARG_HELP,
     callback=_deprecate_options,
 )
+@click.option(
+    "--tags",
+    type=str,
+    default="",
```

Comment:
Is this `default` needed? I see it's in many other options as well so fine to leave it. I'm just curious.

Reply:
`default` provides a value in the case that the flag isn't used - here it needs to be an empty string otherwise we'll get an error in the callback function when a string is expected but `None` is passed through. It eliminates the need to have a `if tags:` statement in the callback function

Reply:
This makes sense, thank you.